wallets/provider-trezor/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { Connect, WalletInfo } from '@rango-dev/wallets-shared';
+
+import { WalletTypes } from '@rango-dev/wallets-shared';
+import TrezorConnect from '@trezor/connect-web';
+import {
+  type BlockchainMeta,
+  evmBlockchains,
+  type SignerFactory,
+} from 'rango-types';
+
+import { getEthereumAccounts, getTrezorInstance } from './helpers';
+import signer from './signer';
+
+const WALLET = WalletTypes.TREZOR;
+const trezorManifest = { appUrl: 'https://rango.exchange/', email: '' };
```

Comment:
appUrl should not be hard coded here. I think you should import it from a config file or another package. Also what if another app try to use this package? how they can set this param?